Soupway/src/cashService/Listener.java

```diff
@@ -0,0 +1,9 @@
+package cashService;
+
+import orders.Order;
+
+public interface Listener {
+	public void notifyForReciept(Order order);
+
+	public void notify(Order order);
```

Comment:
Currently the first method from the listener says that you are notifying for reciept, but second one is not clear.

Some suggestions: 'onOrderPaid(Order order)'

Think for more descriptive name.